content/en/docs/languages/java/instrumentation.md

```diff
@@ -42,6 +42,15 @@ manual instrumentation.
 You don't have to use the example app: if you want to instrument your own app or
 library, follow the instructions here to adapt the process to your own code.
 
+### Prerequisites
+
+For running the example app, ensure that you have the following installed
+locally:
+
+- Java JDK 17+, due to the use of Spring Boot. OpenTelemetry Java only [requires
+  Java 8+][java-vers] otherwise.
```

Comment:
> maybe also "Java JDK 17+" --> "Java 17+"?

That works for me.

As for the Spring Boot version: can we drop it so that we don't have to update the version down the line. The statement remains true without the version: Java 17+ is required _because_ of Spring Boot. WDYT @trask?